sass/elements/_table.scss

```
@@ -16,3 +16,11 @@ tr td {
 tr td:not(:first-child) {
   border-left: 2px solid $subtitle-color;
 }
+
+table tr:first-child th:first-child {
+  border-radius: $border-radius;
+}
+
+table tr:first-child th:last-child {
+  border-radius: $border-radius;
+}
```

Not sure if we want to make all of the corners rounded, just the top left and top right should be enough. Not that it matters that much since it doesn't really make a visual difference with the current styling.
```suggestion
table tr:first-child {
  border-top-left-radius: $border-radius;
  border-top-right-radius: $border-radius;
  
  th:first-child {
    border-top-left-radius: $border-radius;
  }
  
  th:last-child {
    border-top-right-radius: $border-radius;
  }
}
```
Alternatively, just setting `overflow: hidden` for the whole `table` fixes the issue, no extra rounding needed. Of course it would also mean that if we did have actual content overflowing for some reason, it'd get clipped (for better or for worse)

---

Makes sense / looks good on my end.